customize.dist/translations/messages.pl.js

```diff
@@ -0,0 +1,229 @@
+    out.main_p2 = 'Ten projekt wykorzystuje wizualny edytor <a href="http://ckeditor.com/">CKEditor</a> , <a href="https://codemirror.net/">CodeMirror</a>, oraz silnik czasu rzeczywistego <a href="https://github.com/xwiki-contrib/chainpad">ChainPad</a>.';
+    out.main_howitworks = 'Jak to działa';
+    out.main_howitworks_p1 = 'CryptPad wykorzystuje wariant algorytmu <a href="https://en.wikipedia.org/wiki/Operational_transformation">Transformacji operacyjnej</a> który jest wstanie odnaleźć rozdzielony konsensus wprowadzanych danych. Używa do tego <a href="https://bitcoin.org/bitcoin.pdf">Łańcuch blokowy Nakamoto</a>, twór zpopularyzowany przez <a href="https://en.wikipedia.org/wiki/Bitcoin">Bitcoin</a>. W ten sposób algorytm może pominąć potrzebę centralnego serwera do rozwiązywania Konfliktów Operacji Przekształcania poprzez Edycję. Bez potrzeby rozwiązywania konfliktów, serwer może pozostać w niewiedzy o zawartości która jest edytowana w dokumencie.';
+    out.main_about = 'O projekcie';
+    out.main_about_p1 = 'Możesz przeczytać więcej o naszej <a href="/privacy.html" title="">polityce prywatności</a> oraz <a href="/terms.html">warunkach korzystania z usług</a>.';
+
+    out.main_about_p2 = 'Jeżeli masz jakieś pytania lub komentarze, możesz napisać na <a href="https://twitter.com/cryptpad">tweeterze</a>, otworzyć problem na <a href="https://github.com/xwiki-labs/cryptpad/issues/" title="our issue tracker">githubie</a>, przywitać się na ircu (<a href="http://webchat.freenode.net?channels=%23cryptpad&uio=MT1mYWxzZSY5PXRydWUmMTE9Mjg3JjE1PXRydWUe7" title="freenode webchat">irc.freenode.net</a>), lub wysłać nam <a href="mailto:research@xwiki.com">email</a>.';
+
+    out.table_type = 'Typ';
+    out.table_link = 'Link';
+    out.table_created = 'Stworzone';
+    out.table_last = 'Ostatnio otwarte';
+
+    out.button_newpad = 'STWÓRZ PAD WYSIWYG';
+    out.button_newcode = 'STWÓRZ PAD DO KODU';
+    out.button_newpoll = 'STWÓRZ GŁOSOWANIE';
+    out.button_newslide = 'STWÓRZ PREZENTACJĘ';
+
+    // privacy.html
+
+    out.policy_title = 'Polityka prywatności CryptPad';
+    out.policy_whatweknow = 'Co o tobie wiemy';
+    out.policy_whatweknow_p1 = 'Jako aplikacja udostępniana w internecie, CryptPad ma dostęp do metadanych wystawianych przez protokół HTTP. W skład tych danych wchodzi adres IP oraz różne inne nagłówki HTTP które pozwalają na identyfikację twojej przeglądarki. Możesz podejrzeć jakie informacje udostępnia twoja przeglądarka odwiedzając <a target="_blank" rel="noopener noreferrer" href="https://www.whatismybrowser.com/detect/what-http-headers-is-my-browser-sending" title="what http headers is my browser sending">WhatIsMyBrowser.com</a>.';
```

Comment:
As "zobaczyć" is not falls.
You aren't really going to just look at it, but the text is implying that you will just take a peek.

Sticking with "podejrzeć".